core-blocks/latest-comments/editor.scss

```diff
@@ -0,0 +1,16 @@
+.wp-block-latest-comments.has-avatars .avatar {
+	margin-right: 10px;
+}
+
+.edit-post-visual-editor {
```

Comment:
Why do we need this context?

Reply:
I bet we don't; I inherited this CSS and I guess I didn't refactor enough. It can go. 👍